src/mqt/problemsolver/equivalence_checker/equivalence_checker.py

```diff
@@ -0,0 +1,235 @@
+"""This module provides functions to check the equivalence of two circuits using Grover's algorithm."""
```

Comment:
This file should also be renamed to `equivalence_checking` to be consistent.

Reply:
Done